playbooks/deploy_edpm_telemetry_ceilometer.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright 2023 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- hosts: "{{ primary_role_name | default([]) }}:{{ deploy_target_host | default('all') }}"
```

Comment:
should just be "all". We don't need the old TripleO vars (primary_role_name, deploy_target_host)